notebooks/book2/11/mcmc_traceplots_unigauss.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>it may be good to add 1-2 lines of comments on why swap axes is needed. Something like -- expected format by arviz is ... and our current format is ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>